dashboard/test/ui/features/star_labs/applab/versions.feature

```diff
@@ -1,3 +1,5 @@
+# This test has been very flaky recently. Temporarily disabling while we investigate in LABS-867
+@skip
```

Comment:
Since this test is failing for the iPad only (since we disabled phones below), would `@no-mobile` allow us to test on browsers and skip iPad and phone?

Reply:
Oh good point! Updating to `@no_mobile`